lib/modules/manager/velaci/types.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * VelaPipelineConfiguration Types
+ *
+ * Spec: https://github.com/go-vela/types/releases/latest/download/schema.json
+ * Docs: https://go-vela.github.io/docs/reference/yaml/
+ */
+export type Step = { image: string };
+
+export interface VelaPipelineConfiguration {
+  secrets?: {
+    origin?: {
+      image: string;
+    };
+  }[];
+
+  services?: {
```

Comment:
use shared type for step and service.

Reply:
makes sense and found another spot where we can reuse.